css/index.css

```diff
@@ -172,4 +172,62 @@ header .logo {
 footer {
     text-align: center;
     margin: 40px 0 20px;
+}
+
+@media (max-width: 768px) {
+    .container {
+        width: 100%;
+    }
+    .cta {
+
+    }
+}
+
+@media(max-width: 400px) {
```

Comment:
You could probably add another .container class and make it flex to have all of your future elements justified center.